src/components/FlippableHandler.css

```diff
@@ -39,3 +39,8 @@
 #rotation-container > .page:target {
     z-index: 3;
 }
+
+.nq-card.nq-blue-bg {
```

Comment:
This is a fix because any `.nq-card.nq-blue-bg` currently has the style of `.nq-card` and not `.nq-card-bg`.